conf/locale/fr/LC_MESSAGES/django.po

```diff
@@ -27852,3 +27848,13 @@ msgstr "Emplacement du problème: "
 #: lms/templates/instructor/instructor_dashboard_2/data_download.html
 msgid "Download CSV of students Certificates Status"
 msgstr "Télécharger le CSV des étudiants Statut des certificats"
+
+#: Found in edx-theme-microsites and localized
+#: GCS/templates/footer.html
+msgid "Terms of Service and Honour Code"
+msgstr "Conditions d'utilisation et Code d'honneur"
+
+#: Found in edx-theme-microsites and localized
+#: GCS/templates/courseware/course_about.html
+msgid "Enrol Now"
```

Comment:
why the typo on "Enrol" (double l)? is it because they have the same typo on the microsites repo?

Reply:
This is not a typo, this is the source of our problems jaja https://writingexplained.org/enrolment-vs-enrollment-difference British vs American

Reply:
A complete document about this, wauuuuu